src/stories/Modal.js

```diff
@@ -11,6 +12,7 @@ const Container = styled.div`
 class ModalDemo extends React.Component {
   constructor (props) {
     super(props)
+    this.props = props
```

Comment:
I think this is unnecessary and can be removed

Reply:
Nice catch. I was probably testing something and forgot to remove this.